README.md

```diff
@@ -38,6 +38,19 @@ This plugin remains under development and contributions are very welcome, please
 pip install napari-animation
 ```
 
+> [!WARNING]
+> `napari-animation` uses `ffmpeg` to export animations. If you are using a macOS arm64 computer (e.g. M1, M2 processor)
+> the PyPI package does not include the needed binary for your platform. You will need to install `ffmpeg` from using
```

Comment:
```suggestion
> the PyPI package does not include the needed binary for your platform. You will need to install `ffmpeg` using
```

Delete either "from" or "using" at the end of this sentence. Doesn't matter which one, just can't have both.

Reply:
...but since I see we have "from" later on in that sentence, it makes sense to say "using conda, from the conda-forge channel..."